bazel/foreign_cc/llvm.patch

```diff
@@ -0,0 +1,25 @@
+# Workaround for Envoy's CMAKE_BUILD_TYPE=Bazel.
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -247,7 +247,7 @@
+ string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
+ 
+ if (CMAKE_BUILD_TYPE AND
+-    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
++    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL|BAZEL)$")
```

Comment:
Can you upstream this? We want as few patches as possible to improve update velocity.

Reply:
https://github.com/envoyproxy/envoy/issues/12634

Reply:
Same goes for the other patches as well :) We really don't like to accumulate patches; they are easy to write and unblock folks adding new deps, but become a nightmare for maintenance, as it involves costly labor to keep them in sync as we bump version and it impacts our security upgrade velocity as a result.